stable/kommander/Chart.yaml

```diff
@@ -7,4 +7,4 @@ maintainers:
   - name: alejandroEsc
   - name: jimmidyson
 name: kommander
-version: 0.12.4
+version: 0.12.5
```

Comment:
I think we'll want to start a new minor version here and put patch releases in 0.12.x

Reply:
shall this PR go into 1.2? then 0.12.x is correct 👍